jp.yvt.OpenSpades.appdata.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>jp.yvt.OpenSpades.desktop</id>
```

Comment:
is this the actual app-id in the upstream code? This is *not* the recommended format for a flatpak app-id https://github.com/flathub/flathub/wiki/AppData-Guidelines

Reply:
Basically this?
```suggestion
  <id>jp.yvt.OpenSpades</id>
```

Reply:
Yep!